src/components/widgets/exclude-objects/ExcludeObjects.vue

```diff
@@ -29,6 +29,9 @@
         :d="iconCircle"
         class="hitarea"
         @click="$emit('cancel', name)"
+        @touchstart="touchedElement = name"
+        @touchend="handleTouchEnd(name)"
+        @touchcancel="touchedElement = undefined"
```

Comment:
Is it worth also adding `@touchmove="touchedElement = undefined"` so that if moving the canvas it won't raise the cancel dialog?